physics/SFC_Models/Land/Noahmp/noahmpdrv.meta

```diff
@@ -96,6 +96,230 @@
   dimensions = ()
   type = integer
   intent = out
+[land_iau_control]
+  standard_name = land_data_assimilation_control
+  long_name = land data assimilation control
+  units = mixed
+  dimensions = ()
+  type = land_iau_control_type
+  intent = inout
```

Comment:
Since these are declared with the optional attribute, you need to add an `optional = True` line below the intent line for each of these variables.